passive/s3.js

```diff
@@ -1,22 +1,29 @@
 // S3 bucket finder by alishasinghania09@gmail.com
 
-function scan(ps, msg, src) {
-  // populate some parameters which will be needed if s3 bucket url is present
-  var alertRisk = 1;
-  var alertConfidence = 3;
-  var alertTitle = "s3 Bucket URL";
-  var alertDesc = "s3 Bucket URL found in response.";
-  var alertSolution =
-    "Remove s3 Buckets name from response or make sure the permissions in bucket are configured properly.";
-  var cweId = 200;
-  var wascId = 13;
+var ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
 
+function getMetadata() {
+  return ScanRuleMetadata.fromYaml(`
+id: 100035
```

Comment:
Same ID as `Information Disclosure - Java Stack Trace` (also in `scanners.md`).

Reply:
Wow, funny how we all missed that.